skimage/measure/_regionprops.py

```diff
@@ -227,6 +233,20 @@ def euler_number(self):
     def extent(self):
         return self.area / self.image.size
 
+    @property
+    def feret_diameter(self):
+        from ..morphology.convex_hull import convex_hull_image
+        label_image = self._label_image
+        label = self.label
+        identity_convex_hull = convex_hull_image(label_image == label)
```

Comment:
@emmanuelle this is much more quickly implemented with the `.image` property, which contains only the object in question, already in bool form. (It uses `props.slice` to evaluate only in the region area.)